configuration/transitions.md

```diff
@@ -27,9 +27,9 @@ The following transitions are available and executed in order.
 |---|---|
 | maintain_info_document | Records metadata about the document such as when it was replicated. Enabled by default. |
 | update_clinics | Adds a contact's info to a new data record. This is used to attribute an incoming SMS message or report to the appropriate contact. The `rc_code` value on the contact is used to match to the value of the form field set as the `facility_reference` in the [JSON form definition](https://github.com/medic/medic-docs/blob/master/configuration/forms.md#json-forms). This matching is useful when reports are sent on behalf of a facility by unknown or various phone numbers. If `facility_reference` is not set for a form, the contact match is attempted using the sender's phone number. |
-| [registration](#registration) | For registering a patient to a schedule. Performs some validation and creates the patient document if the patient does not already exist. |
+| [registration](#registration) | For registering a patient to a schedule. Performs some validation and creates the patient document if the patient does not already exist. Can create places (as of 3.8.0).|
 | accept_patient_reports | Validates reports about a patient and silences relevant reminders. |
-| [generate_patient_id_on_people](#generate-patient-id-on-people) | Automatically generates the `patient_id` on all person documents. |
+| [generate_patient_id_on_people](#generate-patient-id-on-people) | Automatically generates the `patient_id` on all person documents. As of 3.8.0, automatically generates the `place_id` on all place documents. |
```

Comment:
As this has evolved the name `generate_patient_id_on_people` may not be intuitive anymore. Have we considered renaming this to something like `generate_id_on_contacts`, and deprecating or aliasing `generate_patient_id_on_people`?

Reply:
I have considered it. On other occasions the conclusion was that we will, at some point (4.0), remove deprecated transitions and rename the ones that have legacy names, like "update_clinics". 
I can make this change now, if requested: create a new transition called `generate_shortcode_on_contacts` and move all code from `generate_patient_id_on_people` (sort of like an alias, so the old transition would use the code from the new transition).

Reply:
I think the sooner we use the "good" naming the better, so I would go ahead with that change. We should also warn via medic-conf when the old transition is used in 3.8+ until it is removed fully. If that isn't done via medic/cht-core#5614 we should open an issue in medic-conf.

Reply:
:+1: about updating the transitions in cht-core.

About the medic-conf request, I think that involves a little more work. As far as I can tell, we don't have a mechanism of determining the version of the instance the app settings are to be uploaded to, so that would have to be implemented. 

I created an issue to track this feature: https://github.com/medic/medic-conf/issues/269

Reply:
Perfect, that's great!